Fortinet/fortigate/ingest/parser.yml

```
@@ -240,7 +240,7 @@ stages:
           fortinet.fortigate.attack.name: "{{parsed_event.message.attack}}"
           fortinet.fortigate.attack.id: "{{parsed_event.message.attackid}}"
           fortinet.fortigate.policyid: "{{parsed_event.message.policyid}}"
-          fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid}}"
+          fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid or parsed_event.message.uuid}}"
```

I wonder if we shouldn't move this assignment in a dedicated `set` action with a filter. The `uuid` field is too generic and may contain information that is not related to a policy. I suggest:

```suggestion
          fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid}}"
          
      - set:
          fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid}}"
        filter: "{{parsed_event.message.cfgpath == 'firewall.policy'}}"
```

---

Yes, you're right, I made the change.